v2/client.go

```diff
@@ -0,0 +1,56 @@
+package v2 // import "go.mongodb.org/atlas/mongodbatlas/v2"
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+	"runtime"
+
+	"github.com/mongodb-forks/digest"
+	apiv2latest "go.mongodb.org/atlas/mongodbatlas/v2/api"
+)
+
+const (
+	// CloudURL is default base URL for the services.
+	CloudURL = "https://cloud.mongodb.com/"
```

Comment:
shouldn't this be the default and if you plan to make use this package for tests then i would recommend to make it configurable

Reply:
URL is configurable. Changing name:
```suggestion
	DefaultCloudURL = "https://cloud.mongodb.com/"
```